### ISSUE TITLE
Can't pass some options to vim

### ISSUE THREAD

**Issue by trevordixon** [maebert/jrnl#322]
I'd like to use vim as my editor, but start it with spell checking turned on. I can do `vim -c 'set spell'`, but if I use that command as the editor setting, I get an error when vim starts:

    "spell'" [New File]
    Error detected while processing command line:
    E20: Mark not set

Is it possible to do what I want to currently; am I misunderstanding something? Or is this a jrnl bug? I see `subprocess.call(config['editor'].split() + [tmpfile])` in https://github.com/maebert/jrnl/blob/master/jrnl/util.py#L133. Maybe that `split()` is a naive way of preparing the command, because it shouldn't split in the middle of quotes?

**Comment by philipsd6** [comment]
Why not use [shlex.split](https://docs.python.org/2/library/shlex.html)? It's been around since v1.5.2!
```python
In [1]: import shlex
In [2]: s = "vim -c 'set spell'"
In [3]: shlex.split(s)
Out[3]: ['vim', '-c', 'set spell']
```
&hellip;and it's a hell of a lot nicer to read.

**Comment by pcarranza** [comment]
Much better indeed!

**Comment by maebert** [comment]
Great suggestion @philipsd6, now implemented on 2.0!

**Comment by MinchinWeb** [comment]
@philipsd6 because `shlex.split` breaks on Windows systems... see #348
